src/host/srvinit.cpp

```diff
@@ -439,11 +441,7 @@ try
 
     if (!g.handoffTerminalClsid)
     {
-        TraceLoggingWrite(g_hConhostV2EventTraceProvider,
-                          "SrvInit_ReceiveHandoff_NoTerminal",
-                          TraceLoggingLevel(WINEVENT_LEVEL_VERBOSE),
-                          TraceLoggingKeyword(TIL_KEYWORD_TRACE));
-        return E_NOT_SET;
+        g.handoffTerminalClsid = CLSID_SystemDelegationTerminal;

```

Comment:
interesting, so we're being clever here - we've been invoked for "default console", but we didn't find a delegation terminal, so we can assume that Terminal Stable should be the default terminal, cause the only way to get here is if we're the OpenConsole for a DxD invocation. Makes sense.